OrthoEvol/Orthologs/Blast/blast.py

```diff
@@ -0,0 +1,509 @@
+                                           'taxids': taxon_id}
+                        elif self.method == 2:
+                            update_dict = {'query': query_seq_path,
+                                           'entrez_query': organism}
+                        else:
+                            update_dict = {'query': query_seq_path}
+
+                        self.blastn_log.debug('Blast run has started.')
+                        self.blastn_log.info("Current gene: %s (%s)." %
+                                             (gene, self.tier_dict[gene]))
+                        self.blastn_log.info("Current organism: %s (%s)." % (organism, taxon_id))
+                        # Run Ncbiblastncommandline
+                        self.blastn_wrapper(xml_path=xml_path, gene_path=gene_path,
+                                            parameters=update_dict, gene=gene,
+                                            organism=organism)
+
+                    # If the BLAST has gone through all orthologs then create a
+                    # master accession file.
+                    if gene == genes[-1] and organism == self.org_list[-1]:
+                        self.create_maf()
+                        if self.save_data:
+                            self.post_blast_analysis(self.project)
+                            self.blastn_log.info("Post-blast analysis is complete")
+
+                        # TODO Archive function here
+            self.blastn_log.info("Blast has completed. Check your output.")
+
+
+class OrthoBlastN(BaseBlastN):
+    """OrthoBlastN provides a preconfigured and orthology optimized version
+    of the BaseBlastN class."""
+
+    def __init__(self, project="orthology-gpcr", method=1, template=None,
+                 save_data=True, acc_file="gpcr.csv", copy_from_package=True,
+                 **kwargs):
+        """This class inherits from the BaseBlastN class.
+
+        This class utilizes it's parent classes to search a standalone
+        Blast database for specific orthologs of a gene using a query organism
+        (usually human).  The best hits from the Blast are filtered for the
+        best option in order to get the most accuarate accession numbers for
+        downstream analysis.
+
+        :param project:  The project name (Default: 'orthology-gpcr')
+        :param method: Method used for blasting. (Default: 1)
+        :param template:  The accession file template.
+        :param save_data:  A flag for saving the post_blast data to an excel file.
+        :param acc_file: The accession file to use. (Default: 'gpcr.csv')
+        :param copy_from_package: Copy the acc_file from the package. (Default: True)
+        :param kwargs:
+        """
+        # Set values for methods to prevent using a config.
+        self.taxon_file = None
+        self.__post_blast = True
+        self.project_path = None
+        self.ref_species = 'Homo_sapiens'
```

Comment:
```suggestion
        self.ref_species = ref_species
```

Reply:
I wanted to hard code this class since it's our class for the orthologs. If a user doesn't want that reference species, they can create their own blast with `BaseBlastN`.